xclim/core/units.py

```diff
@@ -153,6 +153,19 @@ def _transform(s):
     if unit == "1":
         unit = ""
 
+    if unit.strip() in [
+        "deg C",
+        "degree C",
+        "degree Celsius",
+        "deg K",
+        "degree K",
+        "degree Kelvin",
+        "deg F",
+        "degree F",
+        "degree Fahrenheit",
```

Comment:
I would include "degrees" in this list, alternatively, we could generate this, ie:
```
deg_expected  = ["deg", "degree", "degrees"]
unit_expected  = ["C", "K", "F", "Celsius", "Kelvin", "Fahrenheit", "kelvin"]  # I've seen kelvin without capitalization before.
list_to_examine = [f"{d} {u}" for d in deg_expected for u in unit_expected]
```

Reply:
Sure, just go ahead and make the change.